### ISSUE TITLE
Logo is not straight

### ISSUE THREAD

**Issue by Izzmo** [oqtane/oqtane.framework#467]
Is there a reason the logo is not straight and skewed a bit? It's not skewed enough to make it seems purposeful and almost seems like it's a mistake.

**Comment by thabaum** [comment]
Which theme are you using?  Some thing can be adjusted can you give us a screenshot and browser you are using?

For quick sharing of screenshots in windows 10 you can use the **Windows Key + SHIFT + S** combination take a screen snippet (cropped to content you desire to share) to memory and **Control + V** to paste.  The we can see what you are dealing with.

**Comment by Izzmo** [comment]
Here's an example:

![image](https://user-images.githubusercontent.com/229083/82757564-dd3b3b80-9da6-11ea-88b5-dae29ce6b0e3.png)

**Comment by thabaum** [comment]
OK I see what you are saying.... picture worth a thousand words.

Here I was creating a few PR's to address another issue, This is the current design.

I thought you might of been talking about how the logo in the theme is offset lower in the oqtane theme.  Which I centered it to submit as a possible PR but I am not working on this... @sbwalker would be to talk to here.

I am going to submit a few PR's to see if the theme gets tuned up along with the framework to work with bootstrap better.

Just an idea because it looks off a bit in the horizontal menu.

![image](https://user-images.githubusercontent.com/13577556/82782645-abab8a00-9e11-11ea-8088-73574c774e50.png)

This is what I was thinking you had been talking about... if you notice my screenshot I have adjusted the image and framework to allow bootstrap navbar to work as described on the bootstrap site.

I am going to submit the PRs here tonight I think as I am done with the research and dev part of this aspect however I noticed the same thing as you did as well.   The Oqtane Icon is a bit taller by one block over the t and under the p so it all is actually centered.
